### PR TITLE
Re enable the provider messaging spec

### PR DESCRIPTION
**Pull request by CatalinVoineag**
This has been skipped until we got a new test key for the widget. This is a sandbox key, not production

### CODE REVIEW

File: .env.test [hunk 1]
```diff
@@ -13,4 +13,4 @@ GOOGLE_TAG_MANAGER_APPLY=GTM-APPLY-XX
 GOOGLE_TAG_MANAGER_MANAGE=GTM-MANAGE-XX
 BULLET=false
 COVERAGE=false
-ZENDESK_MESSAGING_KEY=bc5a052c-d446-45f6-b2d7-b6a90de7e250
+ZENDESK_MESSAGING_KEY=9b56d751-c185-4ccd-947b-e7d603b236fe
```
Comment (dcyoung-dev) [inline .env.test:16]: is this a real key 🙈

Comment (CatalinVoineag) [reply .env.test:16]: Yes. It's a sandbox key. We need a real key to open the widget.

Comment (dcyoung-dev) [reply .env.test:16]: I'm concerned that this is in our source code and not hidden anywhere 🤔

Comment (CatalinVoineag) [reply .env.test]: This is now a github secret

**Review by dcyoung-dev — APPROVED** [review]

**Pull request merged**